java/ingest/ingest-runner/src/test/java/sleeper/ingest/runner/IngestRecordsLocalStackIT.java

```diff
@@ -59,7 +60,7 @@ public void shouldWriteRecordsCorrectly() throws Exception {
         //  - Local files should have been deleted
         assertThat(Paths.get(inputFolderName)).isEmptyDirectory();
         //  - Check quantiles sketches have been written and are correct
-        assertThat(SketchesDeciles.fromFile(schema, fileReferences.get(0)))
+        assertThat(SketchesDeciles.fromFile(schema, fileReferences.get(0), new LocalFileSystemSketchesStore()))
```

Comment:
This seems to be using the wrong sketches store, which suggests that IngestRecordsLocalStackITBase is misconfiguring how the file should be written.

This test is passing, which means it must be writing to the local file system when it shouldn't, since this is a LocalStack test.